modules/cli/src/main/scala/scala/cli/commands/util/VerbosityOptionsUtil.scala

```diff
@@ -5,6 +5,7 @@ import scala.build.interactive.Interactive._
 
 object VerbosityOptionsUtil {
   implicit class VerbosityOptionsOps(v: VerbosityOptions) {
-    val interactiveInstance = if (v.interactive.getOrElse(false)) InteractiveAsk else InteractiveNop
+    def interactiveInstance(forceEnable: Boolean = false) =
+      if (v.interactive.getOrElse(false) || forceEnable) InteractiveAsk else InteractiveNop
```

Comment:
Why not make the parameter the default value?
```suggestion
      if (v.interactive.getOrElse(forceEnable)) InteractiveAsk else InteractiveNop
```

Reply:
If forced, the command-line option is simply ignored, while this way, users can override the default.

Reply:
Makes sense, I applied your suggestion